### PR TITLE
refactor: setup config style

### PR DESCRIPTION
**Pull request by henryiii**
Should be ready, I think.

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
Thanks for this! It's looking good and almost ready.

**Comment by eduardo-rodrigues** [comment]
Hi @henryiii, will you finalise on this one? Seems it's just a step away.

**Comment by eduardo-rodrigues** [comment]
Basically, yes. If I compare with `https://github.com/scikit-hep/particle/blob/master/pyproject.toml` I see some bits missing. At least the part related to the version is important, no?

**Comment by eduardo-rodrigues** [comment]
It seems ready to go. Feel free to merge if happy, @henryiii.

File: skhep/_version.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__version__ = '1.1.0'
```
Comment (henryiii) [inline skhep/_version.py:1]:
```suggestion
__version__ = '1.2.1'
```

**Pull request merged**